pages/buttonexample/index.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import reactCSS from "reactcss";
+import { SketchPicker } from "react-color";
+
+class ButtonExample extends React.Component {
```

Comment:
i dont know if this is the way we learned to work . do you really know how class components are working ?Because they work different to the way we learned functions with states .. class components have a fixed state how far i know .and maybe that can leave to problems and understanding later in your app .